src/app/exams/exams-question.component.html

```diff
@@ -18,7 +18,7 @@
     <mat-error><planet-form-error-messages [control]="questionForm.controls.body"></planet-form-error-messages></mat-error>
   </mat-form-field>
   <div>
-    <button type="button" mat-button (click)="addChoice()" class="margin-lr-5" *ngIf="questionForm.controls.type.value.includes('select')">Add Choice</button>
+      <button type="button" mat-stroked-button (click)="addChoice()" class="margin-lr-5" *ngIf="questionForm.controls.type.value.includes('select')" mat-stroked-button type="button">Add Choice</button>
```

Comment:
Added an extra indent and `mat-stroked-button` is repeated.  Please remove the second one.